go-grpc-gateway/go.mod

```diff
@@ -3,20 +3,22 @@ module github.com/mobilecoinfoundation/mobilecoin/go-grpc-gateway
 go 1.19
 
 require (
-	github.com/golang/glog v1.0.0
-	github.com/golang/protobuf v1.5.2
-	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.1
-	google.golang.org/grpc v1.53.0
-	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0
-	google.golang.org/protobuf v1.28.1
+	github.com/golang/glog v1.2.0
+	github.com/golang/protobuf v1.5.4
```

Comment:
I bumped everything

from https://github.com/mobilecoinfoundation/mobilecoin/security/dependabot/128
```
go: module github.com/golang/protobuf is deprecated: Use the "google.golang.org/protobuf" module instead.
```
I got sidetracked by the deprecation warning thinking it was what was causing dependabot to fail, and lost sight of the actual security bump

Dependabot fails due to the generated code being missing from, https://github.com/mobilecoinfoundation/mobilecoin/blob/main/go-grpc-gateway/build.sh